docs/modules/ROOT/pages/explanations/decisions/multi-team-alert-routing-base-alerts.adoc

```diff
@@ -0,0 +1,71 @@
+= Multi-team alert routing for base alerts
+
+== Problem
+
+Currently, all base alerts configured through component openshift4-monitoring are routed to the same team.
+However, for some alerts (such as KubeJobFailed), this isn't always correct, since there might be jobs that are managed by other teams.
+It would be better to route those alerts dynamically based on some criteria, such as a namespace label.
+
+=== Goals
+
+* Selectively route base alerts to the appropriate teams based on namespace ownership
+* Define how "namespace ownership" can be either configured or inferred
+
+=== Non-Goals
+
+* Define alert routing for custom alerts
+* Handle alerts in namespaces that aren't syn-managed
+
+== Proposals
+
+=== Add namespace labels to relevant alerts
+
+One option is to modify the alert queries to join the `kube_namespace_labels` metric to the result, such that the information therein can be used in the (statically defined) routing rules.
+This implies that namespace ownership is configured by setting a specific label on the namespace in question.
+No suitable label currently exists, so all components would need to be updated to provide one, for example based on component ownership.
+
+However, not all alert rules have the namespace label, since not all are namespace scoped. Blindly joining the namespace label metric will therefore lead to some broken alerts.
```

Comment:
```suggestion
However, not all alert rules have the namespace label, since not all are namespace scoped.
Blindly joining the namespace label metric will therefore lead to some broken alerts.
```